src/main.cpp

```diff
@@ -2139,24 +2247,15 @@ void on_light() {
     }
 }
 
-// Destroy Block
-void on_left_click() {
-    State *s = &g->players->state;
-    int hx, hy, hz;
-    int hw = hit_test(0, s->x, s->y, s->z, s->rx, s->ry, &hx, &hy, &hz);
-    if (hy > 0 && hy < 256 && is_destructable(hw)) {
-        if (hw == TNT) {
-            explode(hx, hy, hz);
-        }
-        set_block(hx, hy, hz, 0);
-        record_block(hx, hy, hz, 0);
-        if (is_plant(get_block(hx, hy + 1, hz))) {
-            set_block(hx, hy + 1, hz, 0);
-        }
-    }
+// Destroy block at given x, y, z.
+extern "C"
+void destroyBlock(int x, int y, int z) {
```

Comment:
So this seems out of date from my last TNT related PR. I think that's where the merge conflict comes from. You can take a crack at trying to fix it, or we can hold off until I pull TNT related functions into separate source/header files. Up to you.

Reply:
I think you should update the code to reflect your most recent pull request to the dev branch. I'll make the same changes to my CluodBranch after the fact.